circuits/node/utils.py

```diff
@@ -13,7 +13,7 @@
 def load_event(s):
     data = json.loads(s)
 
-    name = bytes_to_str(data["name"].encode("utf-8"))
+    name = data["name"]
```

Comment:
So the current code is: https://github.com/circuits/circuits/pull/200/files#diff-015e99aa1b430ebfaa9aa46322a22df0R16 -- I think that fixes it

Essentially the idea is:
 * On Py3 classnames are unicode/text which is what json returns, so there is nothing to do
 * On Py2 classnames need to be bytes so we encode via utf-8